ingester/ingester.go

```diff
@@ -165,6 +174,7 @@ func New(cfg Config, chunkStore cortex_chunk.Store) (*Ingester, error) {
 			Help: "The total number of samples returned from queries.",
 		}),
 	}
+	prometheus.MustRegister(i)
```

Comment:
> Have you written some new tests? Wondering why I didn't hit it.

Yeah, I hit this when writing a second ingester test for the series limiting.

> Global please.

Ok, will get to it next week.

FWIW, with having metrics globally, the concern is less about global variable namespace pollution, but about anyone wanting to reuse a package getting globally registered metrics forced on them. As long as we don't care about reuse, that's fine.